### PR TITLE
Extracted kvutils Config to //leger/ledger-runner-common [DPP-865]

### PR DESCRIPTION
**Pull request by tudor-da**
This PR:
* extracts the `Config` stack in a separate package `ledger-runner-common` since it is used by both Sandbox-on-X and ledger-on-memory. 
* the `DumpIndexMetadata` is extracted as well, since it's also shared.  

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox-on-x/src/app/resources/logback.xml [hunk 1]
```diff
@@ -1,9 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-  ~ Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```
Comment (mziolekda) [inline ledger/sandbox-on-x/src/app/resources/logback.xml:3]: Unintended?

**Review by mziolekda — APPROVED** [review]
I welcome this change wholeheartedly. Next step would be to move the Runner, the common logback etc.

File: ledger/sandbox-on-x/src/app/resources/logback.xml (hunk 1 above)
Comment (tudor-da) [reply ledger/sandbox-on-x/src/app/resources/logback.xml:3]: Right.

**Comment by tudor-da** [comment]
> I welcome this change wholeheartedly. Next step would be to move the Runner, the common logback etc.

Thanks. Not sure what do you mean by that your last sentence. The common logback is in `//ledger/sandbox-common` and the `Runner` is ledger-specific (e.g. the kvutils `Runner` is specific for sandbox-next and `SandboxOnXRunner` for sandbox-on-x. Happy to expand in further PRs if we deem it necessary

**Pull request merged**